### PR TITLE
Homework for variable-declarations_by Norba Tamara

### PR DESCRIPTION
**Pull request by NorbaTamara**

### CODE REVIEW

File: Exercises/variables-declaration/variables-declaration.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+    let incapsulatedVarible;
+}
+
+// Declare a variable which will appear in global object.
+var globalVariable;
+
+// Declare two variables which will be same-names but contain different values.
+var sameName = 5;
+var sameName = 10;
```
Comment (Mike-Savin) [inline Exercises/variables-declaration/variables-declaration.js]: Two variables with different names should contain different values at the same time. In your case second defined variable will overwrite the first one.

Comment (Mike-Savin) [reply Exercises/variables-declaration/variables-declaration.js]: Good work in general

**Review by Mike-Savin — CHANGES_REQUESTED** [review]

**Review by lizarusi — CHANGES_REQUESTED** [review]
All good, just see @Mike-Savin comments

**Comment by NorbaTamara** [comment]
Thank you :) 
Resolved.

**Pull request merged**